tools/changelog/ss13_genchangelog.py

```diff
@@ -1,12 +1,14 @@
-#!/usr/bin/env python2
```

Comment:
You should still keep this shebang, change it to python3 instead of removing it.

Reply:
Many distros no longer ship `env`, so I'm looking for a better alternative shebang.